### DISCUSSION TITLE
Runs successfully but not creating Collections?

### DISCUSSION THREAD

**Discussion by dominic-wood** [meisnate12/Plex-Meta-Manager#69] [Q&A] [answered]
I may just be missing something completely obvious here... I've mapped everything up correctly in the config, and when i run: 

`sudo python3 plex_meta_manager.py --config /config/config.yml --run`

I get no errors and it returns Finished Run, but no collections have been created. 

![Screenshot 2021-02-25 at 23 18 09](https://user-images.githubusercontent.com/79326609/109232378-2bf78500-77c0-11eb-8c9c-7c1e58fb7ad6.png)

I am only testing it out at the moment with 1 collection: 

collections:
Reddit Top 250:
    trakt_list: https://trakt.tv/users/jay-greene/lists/reddit-top-250-2019-edition

Any ideas on what I'm doing wrong?

**Comment by meisnate12** [comment, marked as answer]
this a bug #55 the fix is in dev but its because the program can't locate your YAML Metadata file
